File: src/ui/asq-ui/src/app/_models/domain/video-summary-dto.ts
import { BaseDomainDto } from './base-domain-dto';

export class VideoSummaryDto extends BaseDomainDto{
    public title: string = '';
    public description: string = '';
    public creationUserUniqueId: string = null;
    public creationUserName: string = '';
    public creationUserSurname: string = '';
    public creationUserUsername: string = '';
    public thumbnailUrl: string = '';
    public creationUserThumbnailUrl: string = '';
    public url: string = '';

    constructor(){ super(); }
}

File: src/lib/models/DTO/Domain/VideoSummaryDto.cs
namespace DTO.Domain
{
    public class VideoSummaryDto: BaseDomainDto
    {
        public string Title{ get; set; }
        public string Description{ get; set; }
        public System.Guid CreationUserUniqueId{ get; set; }
        public string CreationUserName{ get; set; }
        public string CreationUserSurname{ get; set; }
        public string CreationUserUsername{ get; set; }
        public string ThumbnailUrl{ get; set; }
        public string CreationUserThumbnailUrl{ get; set; }
        public string Url{ get; set; }
    }
}
